content-script-register/README.md

```diff
@@ -1,24 +1,26 @@
-# User script
+# Content script registration
 
 This extension demonstrates the `browser.contentScripts.register()` API, which enables an extension to register URL-matching content scripts at runtime.
 
+If you want functionality to load user scripts, scripts that can be provided access to extension APIs not available to content scripts, please refer to the [user script registration](https://github.com/mdn/webextensions-examples/tree/master/user-script-register) example.
```

Comment:
I think this is misleading:

> If you want functionality to load user scripts, **scripts that can be provided access to extension APIs not available to content scripts**

This suggests to me that the key difference between content scripts and user scripts is that an extension can give extra powers to user scripts - specifically, "access to extension APIs not available to content scripts".

From my reading of https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/manifest.json/user_scripts, it's **not the case** that user scripts can give this kind of extra access, because the API script, that can essentially provide extra APIs to user scripts:

> has access to the same subset of WebExtension APIs usually available in a content script.

It might be better to say something like:

> The contentScripts.register() API is intended to enable an extension to register at runtime scripts that are packaged with the extension itself. If you want to register third-party user scripts, you should instead use the [`userScripts`](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/userScripts) API. Please refer to [user-script-register](https://github.com/mdn/webextensions-examples/tree/master/user-script-register) for an example of that API.

Reply:
Good point, amended